pilot/pkg/networking/core/v1alpha3/listener_builder.go

```diff
@@ -188,6 +280,9 @@ func (builder *ListenerBuilder) buildVirtualInboundListener(env *model.Environme
 		UseOriginalDst: proto.BoolTrue,
 		FilterChains:   newInboundPassthroughFilterChains(env, node),
 	}
+	if builder.useInboundFilterChain {
```

Comment:
as stated above, nuke this and make this hte only way to construct the virtualInbound.. else we have too many options..

Reply:
To be clear: based on the iptables style, pilot will still generate either old-style virtual inbound, or new style filter-chain for inbound.